kitsune/users/jinja2/users/auth.html

```diff
@@ -15,7 +15,7 @@
 {% if aaq_context and aaq_context.key %}
 <section class="sumo-page-section question-masthead shade-bg">
   <div class="mzp-l-content">
-    {{ progress_bar(3, product_key=aaq_context.key) }}
+    {{ progress_bar(3, product_slug=aaq_context.key) }}
```

Comment:
Same here. I think this should instead be `product_slug=aaq_context.product_slug`?